enterprise/server/gcplink/gcplink.go

```diff
@@ -146,3 +149,106 @@ func makeTokenExchangeRequest(refreshToken string) (string, error) {
 type accessTokenResponse struct {
 	AccessToken string `json:"access_token"`
 }
+
+func GetGCPProject(env environment.Env, ctx context.Context, request *gcpb.GetGCPProjectRequest) (*gcpb.GetGCPProjectResponse, error) {
+	groupID := ""
+	if reqCtx := requestcontext.ProtoRequestContextFromContext(ctx); reqCtx != nil {
+		groupID = reqCtx.GetGroupId()
+	}
```

Comment:
is there a reason this isn't GetGroupID()? is it because these requests ignore the group picker

Reply:
Na, I think I just cribbed it from the wrong place - will fix.

Reply:
Fixed